dojo/templates/dojo/problem_findings_list_snippet.html

```diff
@@ -645,6 +657,53 @@ <h3 class="has-filters">
             });
     });
     </script>
+    <script type="application/javascript">
```

Comment:
Is this the exact same we have in the findings list? If so, does it make sense to have this in a separate file that is included by all views? This way we avoid code duplication.